jest.setup.js

```diff
@@ -36,14 +40,14 @@ jest.mock('Animated', () => {
     timing: (value, config) => ({
       start: (callback) => {
         value.setValue(config.toValue);
-        callback && callback();
+        if (callback) callback();
```

Comment:
for some reason my linter was yelling at me. before we merge this we can see why my linter doesn't match everyone elses

Reply:
Looks like I get a lint error when it tried to put it back. Maybe it's just how our `eslintrc` file is set up? Not sure why it would be different from cores, but ...
![Screenshot 2020-01-29 14 25 32](https://user-images.githubusercontent.com/3229463/73389965-38e73380-42a3-11ea-9273-d66fd4cf2c78.png)

Reply:
I'll work on getting on the linter stuff put back

Reply:
ok so the short version is I think this stuff isn't worth worrying about. yarn and lerna are make our eslint plugin versions not match exactly right even though the `package.json` does match. long term the goal for the core package is not have anything in it anyway. tooling will ultimately be up to us anyway.

![Screen Shot 2020-01-29 at 3 18 58 PM](https://user-images.githubusercontent.com/2659478/73394076-e6117a00-42aa-11ea-9e70-39cd3fad8052.png)

Reply:
linter doesn't like `callback && callback();` since it is considered stylistically bad to use an expression when you're not doing a variable assignment or something. For example, `const a = callback && callback();` would be fine, but `callback && callback();` is considered unnecessary shorthand for just doing `if (callback) callback();`, which the linter is imposing that the latter is easier to read/understand

Reply:
right, but on core it's not enforced. that's what we're trying to figure out.

Reply:
@redreceipt I think I agree with you. We can look into these later. As long as everything works this is small potatoes. There are bigger things to worry about here. I'm going to resolve these for now.